app/src/main/java/com/truongdc21/quickquotes/database/tag/TagDaoDb.kt

```diff
@@ -4,11 +4,11 @@ import com.truongdc21.quickquotes.data.model.Tag
 
 interface TagDaoDb {
 
-    suspend fun insertTag (tag: String)
+    suspend fun insertTag(tag: String)
 
-    suspend fun updateTag (tag: String , id: Int)
+    suspend fun updateTag(tag: String , id: Int)
```

Comment:
```suggestion
   fun updateTag(tag: String , id: Int)
```
Không dùng library e ei,
remove coroutine hộ anh!

Reply:
@daolq-2712 Dạ hôm bữa trong meeting em có hỏi rồi á a , Việt kêu là cho sử dụng